lib/hammer_cli_katello/repository.rb

```diff
@@ -11,7 +11,7 @@ class ListCommand < HammerCLIKatello::ListCommand
           field :name, _("Product")
         end
         field :content_type, _("Content Type")
-        field :url, _("URL")
+        field :relative_path, _("URL")
```

Comment:
I get this output when listing repos:

```
---|--------------|----------------|--------------|----------------------------------------------------------------
ID | NAME         | PRODUCT        | CONTENT TYPE | URL                                                            
---|--------------|----------------|--------------|----------------------------------------------------------------
1  | puppet forge | puppet product | puppet       | Default_Organization/Library/custom/puppet_product/puppet_forge
2  | puppet2      | puppet product | puppet       | Default_Organization/Library/custom/puppet_product/puppet2     
---|--------------|----------------|--------------|----------------------------------------------------------------
```

should the displayed col name also be changed from URL to relative_path?